src/dwg.spec

```diff
@@ -4086,6 +4105,9 @@ DWG_OBJECT(FIELD)
   //DEBUG_HERE_OBJ
   FIELD_T (evaluation_error_msg, 300);
   Table_Value(value)
+  if (error & ~DWG_ERR_INVALIDTYPE)
```

Comment:
logical error (and below):
why are you checking for all errors but INVALIDTYPE and return INVALIDTYPE then?
I think you need to return error instead.

Reply:
Ooops, I wanted to write `if (error & DWG_ERR_INVALIDTYPE)` (check only DWG_ERR_INVALIDTYPE error)
Ok, it will be `if (error) return error;`